Sources/SwiftProtobuf/Message+JSONAdditions.swift

```diff
@@ -34,6 +34,26 @@ public extension Message {
     return visitor.stringResult
   }
 
+  /// Returns a Data containing the UTF-8 JSON serialization of the message.
+  ///
+  /// Unlike binary encoding, presence of required fields is not enforced when
+  /// serializing to JSON.
+  ///
+  /// - Returns: A Data containing the JSON serialization of the message.
+  /// - Throws: `JSONEncodingError` if encoding fails.
+  func jsonUTF8Data() throws -> Data {
+    if let m = self as? _CustomJSONCodable {
+      let string = try m.encodedJSONString()
+      let data = string.data(using: String.Encoding.utf8)! // Cannot fail!
```

Comment:
I think `_CustomJSONCodable` should encode into a `JSONEncoder` instead of just building and returning a String.  That would be a much better fit for streaming encoding, especially for things like `ListValue` and `Struct`.

But that's for a future PR... ;-)